cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/vis_builder/vis_types/table.spec.js

```diff
@@ -85,9 +85,9 @@ export const removeBucket = (bucket) => {
   });
 };
 
-export const testSplitTables = (dir, num) => {
+export const testSplitTables = (num) => {
   cy.getElementByTestId('visTable')
-    .should('have.class', `visTable ${dir}`.trim())
+    .should('have.class', `visTable`)
```

Comment:
better to do `.should('have.class', 'visTable')` since it is a constant string now that doesn't require any variable interpolation.